php/class-coauthors-guest-authors.php

```diff
@@ -929,7 +929,7 @@ function get_guest_author_by( $key, $value, $force = false ) {
 				}
 				// Ensure we aren't doing the lookup by the prefixed value
 				if ( 'user_login' == $key ) {
-					$value = preg_replace( '#^cap\-#', '', $value );
+					$value = preg_replace( '#^cap\-#', '', sanitize_title( $value ) );
```

Comment:
Do you think `sanitize_title_for_query()` would be a better function to use since it gets passed into `$query` later?